pkg/driver/agent.go

```diff
@@ -65,6 +67,22 @@ func NewNode(d *CSIDriver) csi.NodeServer {
 		}
 	}()
 
+	// start the backup controller
+	go func() {
+		err := backup.Start(&ControllerMutex, stopCh)
+		if err != nil {
+			klog.Fatalf("Failed to start ZFS volume snapshot management controller: %s", err.Error())
```

Comment:
n_: log message can probably say `volume snapshot backup controller` and the below one as `volume snapshot restore controller`.